Mathlib/Analysis/CStarAlgebra/Basic.lean

```diff
@@ -5,7 +5,6 @@ Authors: Frédéric Dupuis
 -/
 import Mathlib.Analysis.Normed.Group.Hom
 import Mathlib.Analysis.Normed.Module.Basic
-import Mathlib.Analysis.Normed.Operator.LinearIsometry
```

Comment:
What does `Operator` refer to in this module name? The file doesn't seem to mention the operator norm...